### PR TITLE
feat: add AdExtent to Marketing.yaml

### PR DESCRIPTION
**Pull request by pizofreude**
AdExtent is one of the project involving Tal Keinan, a well-know Zionist figure in the Israeli tech and venture capital scene also former Lieutenant Colonel in Israel Air Force Reserves. He involved in many venture capital firm: Singular, Clarity Capital, KCPS Clarity that intertwined with many Hasbara and dehumanisation of Palestinians.

### CODE REVIEW

File: dataset/companies/Marketing.yaml
Comment (liranelisha) [inline dataset/companies/Marketing.yaml:1]: you are missing description to the entire file and to each part

Comment (pizofreude) [reply dataset/companies/Marketing.yaml:1]: Can you elaborate? I add these:
- Name: "AdExtent"
  Description: "AdExtent helps eCommerce sites turn more visitors to shoppers with personalized product marketing."
  Website: "https://adextent.com/"
  Alternatives: [https://www.thetradedesk.com/]

to dataset/companies/Marketing.yaml

Any further details I should add to the PR?

**Review by abarmawi — CHANGES_REQUESTED** [review]

File: dataset/companies/Marketing.yaml [hunk 1]
```diff
@@ -66,3 +66,7 @@
   Description: ""
   Website: "https://affogata.com/"
   Alternatives: []
+- Name: "AdExtent"
```
[the remark this answers is not in the document]
Comment (pizofreude) [reply dataset/companies/Marketing.yaml:69]: It should be emptied out? That's what I understood from README.

File: dataset/companies/Marketing.yaml
Comment (abarmawi) [reply dataset/companies/Marketing.yaml:1]: @liranelisha 
```
Alternatives: 
  - Name: company name
     Website: https://www.thetradedesk.com/
    Description
```

Comment (abarmawi) [reply dataset/companies/Marketing.yaml:1]: you can also run `npm run validate` to check the schema

File: dataset/companies/Marketing.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (abarmawi) [reply dataset/companies/Marketing.yaml:69]: no it should be in this format

```
Alternatives: 
  - Name: company name
     Website: https://www.thetradedesk.com/
     Description: bla bla
```

**Comment by abarmawi** [comment]
I have added the change to the marketing manually

**Pull request closed without merging**